extensions/wikia/Wall/WallRelatedPages.class.php

```diff
@@ -115,20 +115,20 @@ function getMessageRelatetMessageIds( $messageId ) {
 	 * @param array $messageId
 	 */
 
-	function getMessagesRelatedArticleIds( $messageIds, $orderBy = 'order_index', $db = DB_SLAVE ) {
+	function getMessagesRelatedArticleIds( $messageIds, $orderBy = 'order_index', $dbType = DB_SLAVE ) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "order_index" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)